apps/ensnode/src/api/index.ts

```diff
@@ -1,16 +1,59 @@
-import { db } from "ponder:api";
+import { db, publicClients } from "ponder:api";
 import schema from "ponder:schema";
+import { ponderMetadata } from "@ensnode/ponder-metadata/middleware";
 import { graphql as subgraphGraphQL } from "@ensnode/ponder-subgraph/middleware";
 import { Hono } from "hono";
+import { cors } from "hono/cors";
 import { client, graphql as ponderGraphQL } from "ponder";
+import packageJson from "../../package.json";
+import {
+  ensNodePublicUrl,
+  getEnsDeploymentChain,
+  ponderDatabaseSchema,
+  requestedPluginNames,
+} from "../lib/ponder-helpers";
 
 const app = new Hono();
 
+// use CORS middleware
+app.use(
+  cors({
+    origin: "*",
+  }),
+);
+
+// use root to redirect to the ENSAdmin website with the current server URL as ensnode parameter
+app.use("/", async (ctx) =>
+  ctx.redirect(`https://admin.ensnode.io/about?ensnode=${ensNodePublicUrl()}`),
+);
+
+// use ENSNode middleware at /metadata
+app.get(
+  "/metadata",
+  ponderMetadata({
+    app: {
+      name: packageJson.name,
+      version: packageJson.version,
+    },
+    env: {
+      ACTIVE_PLUGINS: requestedPluginNames().join(","),
+      DATABASE_SCHEMA: ponderDatabaseSchema(),
+      ENS_DEPLOYMENT_CHAIN: getEnsDeploymentChain(),
+    },
+    db,
+    fetchPrometheusMetrics: () =>
+      fetch(`http://localhost:${process.env.PORT}/metrics`).then((res) => res.text()),
+    // TODO: fix the types
+    // @ts-ignore
+    publicClients,
```

Comment:
This might be a bit of a problem to sort out quickly. Perhaps @shrugs will be able to help me figure this one out.

Reply:
i checked this out locally, removed those lines, and don't get a type error on my end, perhaps solved somehow? i also checked with `pnpm run -r typecheck`

Reply:
Cool, I dropped `node_modules` and had them reinstalled. Now it also works on my end 🚀 

Thank you for checking 🤝